cds/modules/theme/static/js/cds/module.js

```diff
@@ -310,6 +320,53 @@ app.filter('groupDownloadable', function() {
   });
 });
 
+// Sort video subformats descending by height
+app.filter('sortVideosDescending', function() {
+  return function(videos) {
+    return _.orderBy(videos, function(video) {
+      return parseInt(video.tags.height);
+    }, ['desc']);
+  }
+});
+
+/**
+ * Gett all files with types passed as argument
+ * @param {Array} files
+ * @param {Array} types the context type of a file
+ *
+ * ex:
+ * data.files | getFilesByType: ['subtitle']
+ * The function will filter all the files with the 'context_type' 'subtitle'
+ */
+app.filter('getFilesByType', function() {
+  return function(files, types) {
+    if (!Array.isArray(files) || !Array.isArray(types) || !types.length || !files.length) { return files; }
+
+    return files.filter((file) => {
+      return types.includes(file.context_type);
```

Comment:
IE does not implement `includes`, please remove :) Welcome to frontend nightmare! :)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes